scripts/vote_2022_05_03.py

```diff
@@ -0,0 +1,114 @@
+"""
+Voting 03/05/2022.
+
+1. Create role ISSUE_ROLE for TokenManager 0xf73a1260d222f447210581DDf212D915c09a3249
```

Comment:
Probably, it's better to say that we're creating permission for an ISSUE_ROLE of TokenManager assigned  to Voting

Reply:
Thanks, I have implemented your siggestion!